### PR TITLE
fix(docs): change README to show this is a seed

### PR DESCRIPTION
**Pull request by renanrodrigo**
This, and then we change the repository name to say it is a seed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,31 @@
-# Beaker.Next Dashboard
+# Beaker React-Patternfly Seed
```
Comment (StykMartin) [inline README.md:1]: This is really boring :zzz: :sleeping: .
Can we change name to something else? For example `BeakerFly`

File: README.md [hunk 2]
```diff
@@ -1,22 +1,31 @@
...
 
-This repository contains code for the next version of Beaker's Web Interface.
+This repository contains code to serve as base for a react/patternfly project.
```
Comment (StykMartin) [inline README.md:3]: React / PatternFly

File: README.md [hunk 3]
```diff
@@ -1,22 +1,31 @@
...
 
-Currently in early development stage. Any kind of change in code / content / behavior may change anytime.
+It has some predefined structure, boilerplate code and examples to start with. Currently it implements:
```
Comment (StykMartin) [inline README.md:5]: ` and` -> `, and`.
`Currently` -> `Currently,`

File: README.md [hunk 4]
```diff
@@ -1,22 +1,31 @@
...
+- React Hooks in functional components
+- Patternfly 4
+- Typescript
```
Comment (StykMartin) [inline README.md:8]: `TypeScript`

File: README.md (hunk 1 above)
Comment (renanrodrigo) [reply README.md:1]: Sure we do!

File: README.md (hunk 3 above)
Comment (renanrodrigo) [reply README.md:5]: There are two groups of people: the ones who use the Oxford comma, and the ones who don't

**Pull request merged**